owl_wms/nn/normalization.py

```diff
@@ -27,15 +30,14 @@ def forward(self, x):
         x = F.normalize(x, dim = -1)
         return x
 
+
 class QKNorm(nn.Module):
     def __init__(self, dim):
         super().__init__()
 
-        self.q_norm = RMSNorm(dim)
-        self.k_norm = RMSNorm(dim)
-
     def forward(self, q, k):
-        return self.q_norm(q), self.k_norm(k)
+        return layer_norm(q), layer_norm(k)
 
 def LayerNorm(dim):
-    return nn.LayerNorm(dim, elementwise_affine = False)
\ No newline at end of file
+    # TODO: remove and replace with `def layer_norm` above
+    return nn.LayerNorm(dim, elementwise_affine = False)
```

Comment:
Still in `attn.py`. `def layer_norm(...)` is equivalent assuming we're normalizing against the last dim.